briefings_server.py

```diff
@@ -201,6 +236,12 @@ def check_upcoming_talks_and_email():
                 ics_file = make_ics_file(subject, plain, r['date'], public_url)
                 send_email(plain+future_talks_plain, html+future_talks_html, mailing_list_email, subject, cc=[speaker_email, host_email], text_file_att=[('calendar.ics',ics_file,'calendar')])
                 send_email(plain+priv_signup_plain+future_talks_plain, html+priv_signup_html+future_talks_html, priv_mailing_list_email, priv_subject, cc=[speaker_email, host_email], text_file_att=[('calendar.ics',ics_file,'calendar')])
+
+
+                # Send a Tweet
+                tweettext = f'Glad to host {r["speaker"]} of {r["affiliation"]} for an OQE seminar, titled "{r["title"]}".\n\n{public_url}\n\nThe recorded talk will be available.'
```

Comment:
recordings are not always available

Reply:
Ok, I’ll just remove that part of the text for now.

There’s a lot of potential for more complex tweet generation, but it’s probably best to start with a pretty simple message template.

There is support for attaching images to tweets, but we’d need a way to have that data around when send_tweet is called.

Reply:
I concur, as simple as possible sounds good to me